jobs/upload_opensearch_config/templates/bin/upload-config.erb

```diff
@@ -117,22 +117,13 @@ echo "--- License"
 
 echo "--- Creating ISM policy"
 
-rollover_policy=$(curl -XGET ${CA} ${CERT} ${KEY} ${BASEURL}/_plugins/_ism/policies/logs-app-rollover_policy)
-TERM=$(echo "$rollover_policy" | grep -o '"_primary_term":[0-9]*' | sed 's/"_primary_term"://')
-SEQ=$(echo "$rollover_policy" | grep -o '"_seq_no":[0-9]*' | sed 's/"_seq_no"://')
+logs-app-rollover_policy=$(curl -XGET ${CA} ${CERT} ${KEY} ${BASEURL}/_plugins/_ism/policies/logs-app-rollover_policy)
+TERM=$(echo "$logs-app-rollover_policy" | grep -o '"_primary_term":[0-9]*' | sed 's/"_primary_term"://')
+SEQ=$(echo "$logs-app-rollover_policy" | grep -o '"_seq_no":[0-9]*' | sed 's/"_seq_no"://')
 if [[ -n "$TERM" ]]; then
-  curl -XPUT ${CA} ${CERT} ${KEY} "${BASEURL}/_plugins/_ism/policies/logs-app-rollover_policy?if_seq_no=${SEQ}&if_primary_term=${TERM}" -H 'Content-Type: application/json' -d'{"policy":{"description":"Rollover to a new index when age or size becomes our target","default_state":"Rollover","states":[{"name":"Rollover","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"rollover":{"min_index_age":"'${INDEX_AGE}'","min_primary_shard_size":"'${DOCUMENT_SIZE}'","copy_alias":false}},{"read_only":{}}],"transitions":[{"state_name":"Old_index","conditions":{"min_index_age":"30d"}}]},{"name":"Old_index","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"force_merge":{"max_num_segments":1}}],"transitions":[{"state_name":"Delete_index","conditions":{"min_index_age":"365d"}}]},{"name":"Delete_index","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"delete":{}}],"transitions":[]}],"ism_template":[{"index_patterns":["logs-app-*"],"priority":100}]}}'
+  curl -XPUT ${CA} ${CERT} ${KEY} "${BASEURL}/_plugins/_ism/policies/logs-app-rollover_policy?if_seq_no=${SEQ}&if_primary_term=${TERM}" -H 'Content-Type: application/json' -d'{"policy":{"description":"Rollover to a new index when age or size becomes our target","default_state":"Rollover","states":[{"name":"Rollover","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"rollover":{"min_index_age":"1d","min_primary_shard_size":"3gb","copy_alias":false}},{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"read_only":{}}],"transitions":[{"state_name":"Merged_index","conditions":{"min_index_age":"30d"}}]},{"name":"Merged_index","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"force_merge":{"max_num_segments":1}}],"transitions":[{"state_name":"Old_index","conditions":{"min_index_age":"90d"}}]},{"name":"Old_index","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"allocation":{"require":{"box_type":"warm"},"include":{},"exclude":{},"wait_for":false}}],"transitions":[{"state_name":"Delete_index","conditions":{"min_index_age":"365d"}}]},{"name":"Delete_index","actions":[{"retry":{"count":3,"backoff":"exponential","delay":"1m"},"delete":{}}],"transitions":[]}],"ism_template":[{"index_patterns":["logs-app-*"],"priority":100}]}}'
```

Comment:
Any chance we can put the ISM policies in a JSON file for readability?

Reply:
That might be easier